easybuild/easyconfigs/m/MATSim/MATSim-2024.0-GCCcore-13.3.0-Java-21.0.8.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PackedBinary'
+
+name = 'MATSim'
+version = '2024.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://www.matsim.org/'
+description = """MATSim is an open-source framework to implement large-scale agent-based transport simulations."""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['https://github.com/matsim-org/matsim-libs/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-release.zip']
+checksums = ['6d5d9cb81f3895209b101cdda30f40ac38a0557c1dd29c446e2748978f25200d']
+
+dependencies = [
+    ('Java', '21.0.8', '', SYSTEM),
```

Comment:
Please use the Java 21  wrapper instead of specifying a specific bugfix release

(also requires updating filenames)